src/components/Toolbar/Index.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+    StyleSheet,
+    ToolbarAndroid,
+    ToolbarAndroidAction
+} from 'react-native';
+import * as React from 'react';
+import icons, {
```

Comment:
Is `icons` being used?

Reply:
nvm I see it is being used, however, where is "icons" coming from? I only see `images` being exported from the assets?

Reply:
I named the import icons since in this case it was for icons. The reason I named the export images was so that it would make sense in contexts in which the images weren't being used as icons. I guess we would export another dictionary for other types of images.

Reply:
I think the questionable thing happening here is that `images` and `icons` are actually the same thing. As I said in my previous comment, the default export is all that is necessary. Also you can pick and choose what you would like to import from a file. So both `import icons from '../../assets'` and `import { images } from '../../assets'` are perfectly valid statements :grin: